docs/hermes.md

```diff
@@ -34,10 +17,10 @@ The easiest way to enable Hermes is to pass the `--useHermes` flag to `react-nat
 ### Using Hermes in an existing project
 Hermes can be enabled in existing projects with a few edits.
 
-First, set the `UseHermes` property to `true` in the `BuildFlags.props` file in your project's `windows` directory:
+First, set the `UseHermes` property to `true` in the `ExperimentalFeatures.props` file in your project's `windows` directory:

```

Comment:
maybe it would be good to make useHermes behave the same way as useWinUI3: you can set it in react-native.config and it will update the props file

Reply:
My hope is we can get the NuGet package slim enough in the future to include by default. That way there would only be a single switch needed